LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Marcos Javier Riganti
```

Comment:
This should change to Instituto Update

Reply:
The content was generated by the install process. Updated